indico/modules/events/contributions/util.py

```diff
@@ -198,6 +199,11 @@ def get_contribution_ical_file(contrib):
     serializer = Serializer.create('ics')
     return BytesIO(serializer(data))
 
+def is_valid_email(email):
```

Comment:
We already have a function `validate_email` as stated in the original issue:

https://github.com/indico/indico/blob/master/indico/util/string.py#L288